datascience/README.md

```diff
@@ -52,3 +59,41 @@ Returns :
 3      DE      4229.0  biomass  2016-12-23T00:00:00+00:00
 4      DE     19919.0     coal  2016-12-23T00:00:00+00:00
 ```
+
+#### get_exchange(countries, start_date, end_date)
+
+This function returns for each day between `start_date` and `end_date`, the electricity production exchanges for each country defined in `countries`. Data are returned as a [pandas DataFrame](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.html).
+
+Parameters :
+
+- countries (*list*) - a list of country codes to pull.
+- start_date (*string*) - start date. Date format is `YYYY-MM-DD`.
+- end_date (*string*) - end date. Date format is `YYYY-MM-DD`.
+
+Data returned :
+
+- country - country that was passed in the `countries` parameter list.
+- country_exchange - country with which the electricity was exchanged.
+- timestamp - date for which the electricity exchange was computed.
+- value - amount of electricity that was exchanged (in MW). Negative value are export of country to country_exchange. Positive value are import of country from country_exchange.
```

Comment:
instead of "amount of electricity that was exchanged", let's use "power exchanged". Power is a flux (J/s or W) so it might be confusing to talk about "amount of X", which would be energy instead.